homeassistant/components/hydrawise/switch.py

```diff
@@ -85,18 +85,11 @@ def update(self):
         mydata = self.hass.data[DATA_HYDRAWISE].data
         _LOGGER.debug("Updating Hydrawise switch: %s", self._name)
         if self._sensor_type == "manual_watering":
```

Comment:
Please cleanup `mydata.relays[self.data["relay"] - 1]` as suggested in `sensor.py`